### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by maths29**
# 💻  Project requirements covered
- I added any missing JSX elements to the project (without adding extra functionality)
- The final design should match Zeplin's design (this includes all elements + the same styling)
- I  used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -20,6 +22,36 @@
...
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
```
Comment (usorfaitheloho) [inline src/App.css]: Kindly implement the following design changes so your design would look better :

- I'd suggest you remove the `author` inbox field from the `Add new Boook` section 👍 
- please use the right margin and padding for the buttons
- kindly add the user icon located at the top right corner of the page
- Kindly reduce the margin of the `Bookstore cms` as shown in the illustration below

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @maths29  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm.js [hunk 2]
```diff
@@ -29,13 +30,12 @@ const BookForm = () => {
...
         <input type="text" value={title} onChange={(e) => setTitle(e.target.value)} placeholder="Book Name" />
-        <input type="text" value={author} onChange={(e) => setAuthor(e.target.value)} placeholder="Author" />
-        <select value={category} onChange={(e) => setCategory(e.target.value)}>
+        <select className="action-btn" value={category} onChange={(e) => setCategory(e.target.value)}>
           <option value="Action">Action</option>
           <option value="Fiction">Fiction</option>
           <option value="Non-Fiction">Non-Fiction</option>
         </select>
-        <button type="submit" onClick={addNewBook}>Add Book</button>
+        <button type="submit" className="addbook-btn" onClick={addNewBook}>Add Book</button>
```
Comment (DuaneDave) [inline src/components/BookForm.js:38]: While making changes based on the previous request, you accidentally added a border that is unnecessary to the button and it makes it look really different from the template. Kindly fix this in order to have a design that is a copy of the original.


image

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @maths29 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Your navbar looks good and neat

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @maths29, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**